packages/blocks/src/blocks/core-audio.tsx

```diff
@@ -1,9 +1,23 @@
-import React, { type JSX } from 'react';
+import React from 'react';
 import {
 	cn,
 	getClassNamesFromString,
 	getStylesFromAttributes,
 } from '@snapwp/core';
+import { BlockData } from '..';
+
+export interface CoreAudioAttributes extends Record< string, unknown > {
+	autoplay?: boolean;
+	caption?: string;
+	loop?: boolean;
+	preload?: string;
+	src?: string;
+	style?: string;
+}
```

Comment:
Think carefully about what attributes should be optional. It does not make sense to render an Audio block without `src`

Reply:
There is a check in the component:
https://github.com/rtCamp/snapwp/blob/110c3da7de0a6c9a489ae1568863f3f974b90cf4/packages/blocks/src/blocks/core-audio.tsx#L37-L39

I suppose returning null when no src is passed is the expected behaviour, since there is a test for the same:
https://github.com/rtCamp/snapwp/blob/110c3da7de0a6c9a489ae1568863f3f974b90cf4/packages/blocks/src/blocks/tests/core-audio.test.tsx#L66-L77